tests/how_to_test_systems.rs

```diff
@@ -32,14 +32,12 @@ fn despawn_dead_enemies(
     }
 }
 
-fn hurt_enemies(mut enemies: Query<&mut Enemy>) {
-    for mut enemy in &mut enemies {
-        enemy.hit_points -= 1;
-    }
+fn hurt_enemies(mut enemies: Query<&mut Enemy>){
+    enemies.iter_mut().for_each(|mut enemy| enemy.hit_points -= 1);
```

Comment:
This is fine, but I don't think it's a clear improvement when the point of the example is to teach something else.